enterprise/server/remote_execution/dirtools/dirtools.go

```diff
@@ -608,14 +669,19 @@ func (ff *BatchFileFetcher) bytestreamReadFiles(ctx context.Context, instanceNam
 	if err != nil {
 		return err
 	}
-	if err := cachetools.GetBlob(ctx, ff.bsClient, digest.NewResourceName(fp.FileNode.Digest, instanceName), f); err != nil {
+	resourceName := digest.NewResourceName(fp.FileNode.Digest, instanceName)
+	if ff.supportsCompression() {
```

Comment:
same here, why not pass supportsCompression from the caller?

Reply:
I removed the other one / double checking that was was happening before.

I'd prefer to leave this one because the call is only done once, beyond that it's just referencing an instance var, and bytestreamReadFiles already has uh... 5 args.